evaluator/images/clang-tidy/build_urls.py

```diff
@@ -16,9 +19,9 @@ def get(url):
     except FileNotFoundError:
         logging.info("Downloading %s", url)
         try:
-            with urllib.request.urlopen(url) as f:
+            with urlopen(url) as f:
```

Comment:
Somehow, I like the longer version with full package path a bit more. What do you think @Kobzol ?